src/Semantics.js

```diff
@@ -96,6 +96,30 @@ Object.defineProperties(Wrapper.prototype, {
   // Returns the number of children of this CST node.
   numChildren: {get: function() { return this._node.numChildren(); }},
 
+  // Returns the minimum number of children for this CST node, if it's an iterator node.
+  // Otherwise, throws an exception.
+  minNumChildren: {
+    get: function() {
+      if (this.isIteration()) {
+        return this._node.minMatches;
+      }
+      throw new TypeError(
```

Comment:
Is there any scenario where it might make more sense to return 0 here instead of throwing an exception? I can't think of one, just a thought.